components/navigation/nav.tsx

```diff
@@ -52,16 +53,27 @@ const Navigation = () => {
         ))}
       </div>
 
-      <div className="ml-auto flex items-center gap-4 hidden">
-        <Link href="/login">
-          <a className="text-sm font-medium text-gray-700 hover:text-green-500">
-            Logga in
-          </a>
-        </Link>
+      <div className="ml-auto flex items-center gap-4">
+        {/* this we might need to change to false instead of undefined. */}
+        {user?.isLoggedIn === undefined && (
```

Comment:
Logout ska väl vara på samma ställe, vanligt att använda `cond ? () : ()` då,

så {user?.isLoggedIn ? &lt;Logout /> : &lt;Login />}

Reply:
semi fixed, maybe create separate components for each button later.